### DISCUSSION TITLE
if youre trying to apply this tweak to win10, DO NOT TRY.

### DISCUSSION THREAD

**Discussion by MuscularPuky** [valinet/ExplorerPatcher#308] [General]
actually im not sure this was caused by dll patch, but it seems almost certain to me of timing. i-im just thought 10.0.21390.2025 is closer to 10.0.22000 so it will be fine and it was stupid think. windows showed me chkdsk as last resort.

first, explorer patcher trys to find symbols but never find them. OK. so far there are no errors. but in setting app, allmost switches do not work. when i toggle and out and then come back, it is undone. even microsoft defender resets the switch in real time. when i off, turn it back on. when i on, turn it back off. i cant help with this.

second, mmc.exe is blocked by red UAC. services.msc, devices manager, local group policy... everything. in addition, task manager is yellow. FUNNY thing is, it works fine with elevated cmd or powershell.

last, sfc /scannow does not work cause of windows resource protection while dicm is work. i havent tried safe boot yet.

yes. not made dll backup. overwritten.

unfortunately, my C drive's total capacity is 128GB, so i had to turn off system restore... exciting!

i wonder how it will work if i bring 10.0.22000 explorer here but this is my main PC.

of course, maybe its conflict with korean mui. whatever cause, it didnt work in my case. so if you are trying to apply this tweak for windows 11 to windows 10.9, DON'T.

**Comment by valinet** [comment]
We have the “Discussions” section for non-issues, I will move this there.

Also, some tweaks may work on 10 indeed, but one would have to manually port only the relevant parts, the general mode of operation for this app is otherwise specific to running on Windows 11. Build numbers are not an indicator of how close one binary is to a previous or subsequent version of it.

Thanks

**Reply by MuscularPuky** [reply]
oh-my mistaked. thanks for moving. i solved my problem by updating to 10.0.22499. i dont wanted win11... and formatting mp pc either so i did.
